### PR TITLE
adding draft changes

### PR DESCRIPTION
**Pull request by gbidsilva**
Draft PR to discuss the changes required.

### CODE REVIEW

**Comment by garydgregory** [comment]
> Draft PR to discuss the changes required.

Hi @gbidsilva 
Please provide a useful description.

File: src/test/java/org/apache/commons/csv/CSVParserTest.java [hunk 1]
```diff
@@ -1642,4 +1642,24 @@ private void validateRecordPosition(final String lineSeparator) throws IOExcepti
         parser.close();
     }
 
+
+    @Test
+    public void test_empty_columns() throws IOException {
```
Comment (garydgregory) [inline src/test/java/org/apache/commons/csv/CSVParserTest.java:1647]: This is not a test, the output is not validated.

**Review by garydgregory — COMMENTED** [review]
-1 see developer's mailing list.

**Comment by gbidsilva** [comment]
This PR is created just for the issue discussion purposes and this is not a merge ready PR.
In the mailing list, comments have been provided.
Closing the PR.

**Pull request closed without merging**